src/streamers.jl

```diff
@@ -298,6 +298,7 @@ struct TObjArray
     low::Int32
     elements
 end
+Base.getindex(obj::TObjArray, index) = obj.elements[index]
```

Comment:
this ever gets used?

Reply:
Don't think I ended up using it. Only on the repl to make `t["blah"].fBranches.elements[1].fBranches.elements[1]...` into `t["blah"].fBranches[1].fBranches[1]...` when dealing with the nested p4 branches. Let me know and I'll take it out.